### PR TITLE
i18n(zh-cn): translate `\guides\cms\buttercms.mdx` and `\guides\cms\cloudcannon.mdx`

### PR DESCRIPTION
**Pull request by zreren**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

Translate guides/cms/buttercms.mdx content to Chinese.

### CODE REVIEW

File: src/content/docs/zh-cn/guides/cms/buttercms.mdx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+## 与 Astro 集成
+:::note
+要查看完整的博客网站示例，请参阅[Astro + ButterCMS入门项目](https://buttercms.com/starters/astro-starter-project/).。
+:::
+在本节中，我们将使用 ButterCMS SDK 将数据引入你的 Astro 项目中。
+要开始使用，你将需要以下内容：
+### 前置准备
+
+1. **Astro项目** -如果你还没有Astro项目，我们的[安装指南](/zh-cn/install/auto/)将帮助你快速创建并运行一个项目。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/buttercms.mdx:22]:
```suggestion
1. **Astro 项目** -如果你还没有 Astro 项目，我们的[安装指南](/zh-cn/install/auto/)将帮助你快速创建并运行一个项目。
```

File: src/content/docs/zh-cn/guides/cms/buttercms.mdx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+2. **ButterCMS帐户**. 如果你还没有ButterCMS帐户，你可以通过[注册](https://buttercms.com/join/) 免费试用来创建一个。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/buttercms.mdx:24]:
```suggestion
2. ** ButterCMS 帐户**. 如果你还没有 ButterCMS 帐户，你可以通过[注册](https://buttercms.com/join/) 免费试用来创建一个。
```

File: src/content/docs/zh-cn/guides/cms/buttercms.mdx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+3. **你的ButterCMS API令牌** - 你可以在[设置](https://buttercms.com/settings/) 页面找到你的API令牌。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/buttercms.mdx:26]:
```suggestion
3. **你的 ButterCMS API TOKEN** - 你可以在[设置](https://buttercms.com/settings/) 页面找到你的API TOKEN。
```

File: src/content/docs/zh-cn/guides/cms/buttercms.mdx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+### 配置
+
+1. 在你的项目根目录中创建一个名为 .env 的文件，并将你的API令牌作为环境变量添加进去：
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/buttercms.mdx:30]:
```suggestion
1. 在你的项目根目录中创建一个名为 `.env` 的文件，并将你的 API TOKEN 作为环境变量添加进去：
```

File: src/content/docs/zh-cn/guides/cms/buttercms.mdx [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+
+```ini title=".env"
+BUTTER_TOKEN=YOUR_API_TOKEN_HERE
+```
+
+:::tip
+了解更多关于[使用环境变量](/zh-cn/guides/environment-variables/)和Astro中的 .env 文件的信息。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/buttercms.mdx:37]:
```suggestion
了解更多关于[使用环境变量](/zh-cn/guides/environment-variables/)和 Astro 中的 `.env` 文件的信息。
```

File: src/content/docs/zh-cn/guides/cms/buttercms.mdx [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+:::
+
+2. 将ButterCMS SDK安装为依赖项：
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/buttercms.mdx:40]:
```suggestion
2. 将 ButterCMS SDK 安装为依赖项：
```

File: src/content/docs/zh-cn/guides/cms/buttercms.mdx [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add buttercms
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+3. 在你的项目中创建一个新的 src/lib/ 目录，并在其中创建一个名为 buttercms.js 的文件
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/buttercms.mdx:59]:
```suggestion
3. 在你的项目中创建一个新的 `src/lib/` 目录，并在其中创建一个名为 `buttercms.js` 的文件
```

File: src/content/docs/zh-cn/guides/cms/buttercms.mdx [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+
+```js title="src/lib/buttercms.js"
+import Butter from "buttercms";
+
+export const butterClient = Butter(import.meta.env.BUTTER_TOKEN);
+```
+
+这将使用你的API令牌对SDK进行身份验证，并将其导出供整个项目使用。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/buttercms.mdx:67]:
```suggestion
这将使用你的 API TOKEN 对 SDK 进行身份验证，并将其导出供整个项目使用。
```

File: src/content/docs/zh-cn/guides/cms/buttercms.mdx [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+
+### 获取数据
+要获取内容，导入这个客户端并使用其中的一个 retrieve 函数。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/buttercms.mdx:70]:
```suggestion
要获取内容，导入这个客户端并使用其中的一个 `retrieve` 函数。
```

File: src/content/docs/zh-cn/guides/cms/buttercms.mdx [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+
+在这个例子中，我们将会[检索一个集合](https://buttercms.com/docs/api/#retrieve-a-collection)，
+该集合有三个字段：一个短文本字段 `name`，一个数字字段 `price` 和一个WYSIWYG的富文本字段 `description`。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/buttercms.mdx:73]:
```suggestion
该集合有三个字段：一个短文本字段 `name`，一个数字字段 `price` 和一个 WYSIWYG 的富文本字段 `description`。
```

File: src/content/docs/zh-cn/guides/cms/buttercms.mdx [hunk 11]
```diff
@@ -0,0 +1,131 @@
...
+<body>
+	{items.map(item => <div>
+		<h2>{item.name} - ${item.price}</h2>
+		<p set:html={item.description}></p>
+	</div>)}
+</body>
+```
+
+
+这个接口反映了字段的类型。WYSIWYG富文本字段 `description` 加载为HTML字符串，而 [`set:html`](/zh-cn/reference/directives-reference/#sethtml) 指令将 HTML 字符串注入元素中进行渲染。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/buttercms.mdx:97]:
```suggestion
这个接口反映了字段的类型。WYSIWYG 富文本字段 `description` 加载为 HTML 字符串，而 [`set:html`](/zh-cn/reference/directives-reference/#sethtml) 指令将 HTML 字符串注入元素中进行渲染。
```

File: src/content/docs/zh-cn/guides/cms/buttercms.mdx [hunk 12]
```diff
@@ -0,0 +1,131 @@
...
+  <body>
+    <h1>{fields.headline}</h1>
+    <img src={fields.hero_image} />
+  </body>
+</html>
+```
+
+## 官方资源
+
+- [Astro + ButterCMS入门项目](https://buttercms.com/starters/astro-starter-project/)
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/buttercms.mdx:126]:
```suggestion
- [Astro + ButterCMS 入门项目](https://buttercms.com/starters/astro-starter-project/)
```

File: src/content/docs/zh-cn/guides/cms/buttercms.mdx [hunk 13]
```diff
@@ -0,0 +1,131 @@
...
+- [ButterCMS API文档](https://buttercms.com/docs/api/)
+- ButterCMS的[JavaScript指导](https://buttercms.com/docs/api-client/javascript/)
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/buttercms.mdx:128]:
```suggestion
- ButterCMS 的 [JavaScript 指南](https://buttercms.com/docs/api-client/javascript/)
```

File: src/content/docs/zh-cn/guides/cms/buttercms.mdx [hunk 14]
```diff
@@ -0,0 +1,131 @@
...
+    <h1>{fields.headline}</h1>
+    <img src={fields.hero_image} />
+  </body>
+</html>
+```
+
+## 官方资源
+
+- [Astro + ButterCMS入门项目](https://buttercms.com/starters/astro-starter-project/)
+- [ButterCMS API文档](https://buttercms.com/docs/api/)
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/buttercms.mdx:127]:
```suggestion
- [ButterCMS API 文档](https://buttercms.com/docs/api/)
```

File: src/content/docs/zh-cn/guides/cms/cloudcannon.mdx [hunk 15]
```diff
@@ -0,0 +1,22 @@
+---
+title: CloudCannon与Astro
+description: 使用CloudCannon作为CMS为你的Astro项目添加内容
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/cloudcannon.mdx:3]:
```suggestion
description: 使用 CloudCannon 作为 CMS 为你的 Astro 项目添加内容
```

File: src/content/docs/zh-cn/guides/cms/cloudcannon.mdx [hunk 16]
```diff
@@ -0,0 +1,22 @@
...
+type: cms
+stub: true
+service: CloudCannon
+i18nReady: true
+---
+
+[CloudCannon](https://cloudcannon.com)是一个基于Git的内容管理系统，为你的内容提供了一个可视化的编辑器。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/cloudcannon.mdx:10]:
```suggestion
[CloudCannon](https://cloudcannon.com) 是一个基于 Git 的内容管理系统，为你的内容提供了一个可视化的编辑器。
```

File: src/content/docs/zh-cn/guides/cms/cloudcannon.mdx [hunk 17]
```diff
@@ -0,0 +1,22 @@
...
+
+## 官方资源
+
+- [Astro入门指南](https://cloudcannon.com/documentation/guides/astro-starter-guide/)
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/cloudcannon.mdx:14]:
```suggestion
- [Astro 入门指南](https://cloudcannon.com/documentation/guides/astro-starter-guide/)
```

File: src/content/docs/zh-cn/guides/cms/cloudcannon.mdx [hunk 18]
```diff
@@ -0,0 +1,22 @@
...
+- [Bookshop & Astro指南](https://cloudcannon.com/documentation/guides/bookshop-astro-guide/)
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/cloudcannon.mdx:15]:
```suggestion
- [Bookshop & Astro 指南](https://cloudcannon.com/documentation/guides/bookshop-astro-guide/)
```

File: src/content/docs/zh-cn/guides/cms/cloudcannon.mdx [hunk 19]
```diff
@@ -0,0 +1,22 @@
...
+- [Astro初级教程系列](https://cloudcannon.com/tutorials/astro-beginners-tutorial-series/)
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/cloudcannon.mdx:16]:
```suggestion
- [Astro 初级教程系列](https://cloudcannon.com/tutorials/astro-beginners-tutorial-series/)
```

File: src/content/docs/zh-cn/guides/cms/cloudcannon.mdx [hunk 20]
```diff
@@ -0,0 +1,22 @@
...
+- 博客: [CloudCannon的实时编辑如何与Astro和Bookshop一起工作](https://cloudcannon.com/blog/how-cloudcannons-live-editing-works-with-astro-and-bookshop/)
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/cloudcannon.mdx:17]:
```suggestion
- 博客: [CloudCannon 的实时编辑如何与 Astro 和 Bookshop 一起工作](https://cloudcannon.com/blog/how-cloudcannons-live-editing-works-with-astro-and-bookshop/)
```

File: src/content/docs/zh-cn/guides/cms/cloudcannon.mdx [hunk 21]
```diff
@@ -0,0 +1,22 @@
...
+- 博客: [为所有Astro用户提供超凡的支持](https://cloudcannon.com/blog/out-of-this-world-support-for-all-astro-users/)
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/cloudcannon.mdx:18]:
```suggestion
- 博客: [为所有 Astro 用户提供超凡的支持](https://cloudcannon.com/blog/out-of-this-world-support-for-all-astro-users/)
```

**Review by liruifengv — CHANGES_REQUESTED** [review]

**Review by liruifengv — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**